src/komal/assignment_14/ShopScanner.java

```diff
@@ -0,0 +1,127 @@
+package komal.assignment_14;
+
+import java.util.Scanner;
+
+public class ShopScanner {
+	int maggie;
+	int dosa;
+	int oilpouch;
+	int panipuri;
+	int masala;
+	
+	void setInitStock() {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Enter the initial stock quantity for maggie: ");
+		maggie = scanner.nextInt();
+		System.out.println("Enter the initial stock quantity for dosa: ");
+		dosa = scanner.nextInt();
+		System.out.println("Enter the initial stock quantity for oilpouch: ");
+		oilpouch = scanner.nextInt();
+		System.out.println("Enter the initial stock quantity for panipuri: ");
+		panipuri = scanner.nextInt();
+		System.out.println("Enter the initial stock quantity for masala: ");
+		masala = scanner.nextInt();
+	}
+		
+	void requestedQty() {
```

Comment:
We should not ask, user to give inputs after calling methods, 
We should ask user to give input while calling methods, and we take input once only from main method not in every method